README.md

```diff
@@ -1,108 +1,45 @@
-# Introduction to Git and Version Control
+# Learn git
```

Comment:
I think its important for version control to be mentioned at least. Git is only one of many tools.